cardano-cli/src/Cardano/CLI/EraBased/Run/Address.hs

```diff
@@ -64,7 +65,7 @@ generateAndWriteByronKeyFiles :: ()
 generateAndWriteByronKeyFiles asType vkf skf = do
   uncurry (writeByronPaymentKeyFiles vkf skf) =<< liftIO (generateKeyPair asType)
 
-generateAndWriteKeyFiles :: ()
+generateAndWriteKeyFiles_ :: ()
```

Comment:
Please remove this function that is polluting the namespace while providing little value: callers can call `void` themselves :+1: